fpga_interchange/examples/devices/xc7a200t/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+generate_xc7_device_db(
+    device xc7a200t
+    part xc7a200tsbg484-1
+    device_target xc7a200t_target
+)
+
+generate_chipdb(
+    family ${family}
+    device xc7a200t
+    part xc7a200tsbg484-1
+    device_target ${xc7a200t_target}
+    bel_bucket_seeds ${PYTHON_INTERCHANGE_PATH}/test_data/series7_bel_buckets.yaml
```

Comment:
This needs to be updated now that #644 merged, see https://github.com/YosysHQ/nextpnr/pull/644/commits/720f64ea6017fa3e42f80eba64cc615c64788914#diff-e8ebf6961ee292d34a9c55086f82687a83189a2846de6c4a1102f391c8a27168R12

Reply:
Fixed